bot/cogs/fun.py

```diff
@@ -87,6 +87,40 @@ async def eight_ball(self, ctx, *, question=None):
                 description="Usage: `.8ball will this command work?`",
             )
             await ctx.send(embed=embed)
+            
+    @command(
+        brief="Play a game of rock paper scissors",
+        description="Enter an option between rock, paper, scissor after .play",
+        aliases= ["play", "Play", "RPS"],
+    )
+    async def rps(ctx, *, response=None):
+        if isinstance(response, str):
```

Comment:
response will always be a string since all inputs from discord are strings, this check can be removed.